spec/schema_enforcer/ensure_uuid_primary_key_has_default_spec.cr

```diff
@@ -23,7 +23,7 @@ describe Avram::SchemaEnforcer::EnsureUUIDPrimaryKeyHasDefault do
   end
 
   it "raises when table's uuid primary key does not have default" do
-    expect_schema_mismatch "Primary key on the 'prices' table has the type set as uuid but does not have a default value." do
+    expect_schema_mismatch "Primary key on the 'unused_uuid_table' table has the type set as uuid but does not have a default value." do
```

Comment:
I had to make a new table to test this because `prices` gets used in other instances, and was throwing some invalid errors for not having an `id` value.